src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/SceneProfilesExplorer.tsx

```diff
@@ -100,7 +111,7 @@ export class SceneProfilesExplorer extends SceneObjectBase<SceneProfilesExplorer
       key: 'profiles-explorer',
       explorationType: undefined,
       body: undefined,
-      $timeRange: new SceneTimeRange({}),
+      $timeRange: new SceneTimeRange(getDefaultTimeRange()),
```

Comment:
Before, we relied on some parts of the app to call `useTimeRangeFromUrl`, which would set the default time range.